data/ctf_platforms/jeopardy.json

```diff
@@ -246,6 +246,14 @@
       "description": "CTF environment setup tools for OWASP Juice Shop; supports CTFd and FBCTF",
       "language": "JavaScript",
       "price": "Free"
+    },
+     {
+      "name": "Parrot CTFs",
+      "website": "https://parrot-ctfs.com",
+      "source": "https://github.com/Parrot-CTFs",
```

Comment:
Source was taken offline due to a few security concerns and the API were suspending accounts because of this so that problem is being resolved the entire source is open source under the MIT license. The platform does a variety of things CTFs, Hosting, Creation, and Business Vetting Stuff. There are contests that go on within the platform.  If you would Like I can move it over to the Challenge Platforms is this still the case?

Reply:
I am going to resolve this and remake it if needed.